### ISSUE TITLE
[Bug] Transaction failed during `GameInstance.Scan` on Mac

### ISSUE THREAD

**Issue by ofbakirci** [KSP-CKAN/CKAN#3770]
Background
----------
* Operating System: macOS 13.1 Ventura 
* CKAN Version: v1.31.2
* KSP Version: 1.25.5.3190

**Have you made any manual changes to your GameData folder (i.e., not via CKAN)?**
No, I have not. 

Problem
-------
When I try to run CKAN, first screen on terminal shows up, says "Press any key to continue", after that error shows up and CKAN crashed back.

Install Mono
Check if Mono install is working using directions [here](https://www.mono-project.com/docs/getting-started/mono-basics/)
Copy CKAN to KSP main folder
Run CKAN
Press any key.

**Expected behavior**
Terminal UI of CKAN showing up

**Screenshots (if applicable)**
[Error screen](https://imgur.com/F9gV4Xd
)

**CKAN error code (if applicable):**

```
Unhandled Exception:
System.Transactions.TransactionAbortedException: Transaction failed ---> System.ArgumentOutOfRangeException: Number must be either non-negative and less than or equal to Int32.MaxValue or -1.
Parameter name: timeout
  at System.Threading.WaitHandle.WaitOne (System.TimeSpan timeout, System.Boolean exitContext) [0x00026] in <e0a3c60063684e309e609c6019ed1f88>:0 
  at System.Transactions.Transaction.DoPreparePhase () [0x00055] in <8bca7f503b7943c0aceeedfbf6372e05>:0 
  at System.Transactions.Transaction.DoCommit () [0x00076] in <8bca7f503b7943c0aceeedfbf6372e05>:0 
  at System.Transactions.Transaction.CommitInternal () [0x00022] in <8bca7f503b7943c0aceeedfbf6372e05>:0 
   --- End of inner exception stack trace ---
  at System.Transactions.Transaction.CommitInternal () [0x00039] in <8bca7f503b7943c0aceeedfbf6372e05>:0 
  at System.Transactions.TransactionScope.Dispose () [0x001ec] in <8bca7f503b7943c0aceeedfbf6372e05>:0 
  at CKAN.GameInstance.Scan () [0x0013d] in <79ae9046bd9c43dabfd2f7be4b883034>:0 
  at CKAN.ConsoleUI.ModListScreen.ScanForMods () [0x0000b] in <79ae9046bd9c43dabfd2f7be4b883034>:0 
  at CKAN.ConsoleUI.ModListScreen.GetAllMods (CKAN.ConsoleUI.Toolkit.ConsoleTheme theme, System.Boolean force) [0x00000] in <79ae9046bd9c43dabfd2f7be4b883034>:0 
  at CKAN.ConsoleUI.ModListScreen..ctor (CKAN.GameInstanceManager mgr, System.Boolean dbg, CKAN.ConsoleUI.Toolkit.ConsoleTheme regTheme) [0x00045] in <79ae9046bd9c43dabfd2f7be4b883034>:0 
  at CKAN.ConsoleUI.ConsoleCKAN..ctor (CKAN.GameInstanceManager mgr, System.String themeName, System.Boolean debug) [0x00081] in <79ae9046bd9c43dabfd2f7be4b883034>:0 
  at CKAN.ConsoleUI.ConsoleUI.Main_ (System.String[] args, CKAN.GameInstanceManager manager, System.String themeName, System.Boolean debug) [0x00005] in <79ae9046bd9c43dabfd2f7be4b883034>:0 
  at CKAN.CmdLine.MainClass.ConsoleUi (CKAN.GameInstanceManager manager, CKAN.CmdLine.ConsoleUIOptions opts, System.String[] args) [0x00034] in <79ae9046bd9c43dabfd2f7be4b883034>:0 
  at CKAN.CmdLine.MainClass.RunSimpleAction (CKAN.CmdLine.Options cmdline, CKAN.CmdLine.CommonOptions options, System.String[] args, CKAN.IUser user, CKAN.GameInstanceManager manager) [0x002df] in <79ae9046bd9c43dabfd2f7be4b883034>:0 
  at CKAN.CmdLine.MainClass.Execute (CKAN.GameInstanceManager manager, CKAN.CmdLine.CommonOptions opts, System.String[] args) [0x00296] in <79ae9046bd9c43dabfd2f7be4b883034>:0 
  at CKAN.CmdLine.MainClass.Main (System.String[] args) [0x000e9] in <79ae9046bd9c43dabfd2f7be4b883034>:0 
[ERROR] FATAL UNHANDLED EXCEPTION: System.Transactions.TransactionAbortedException: Transaction failed ---> System.ArgumentOutOfRangeException: Number must be either non-negative and less than or equal to Int32.MaxValue or -1.
Parameter name: timeout
  at System.Threading.WaitHandle.WaitOne (System.TimeSpan timeout, System.Boolean exitContext) [0x00026] in <e0a3c60063684e309e609c6019ed1f88>:0 
  at System.Transactions.Transaction.DoPreparePhase () [0x00055] in <8bca7f503b7943c0aceeedfbf6372e05>:0 
  at System.Transactions.Transaction.DoCommit () [0x00076] in <8bca7f503b7943c0aceeedfbf6372e05>:0 
  at System.Transactions.Transaction.CommitInternal () [0x00022] in <8bca7f503b7943c0aceeedfbf6372e05>:0 
   --- End of inner exception stack trace ---
  at System.Transactions.Transaction.CommitInternal () [0x00039] in <8bca7f503b7943c0aceeedfbf6372e05>:0 
  at System.Transactions.TransactionScope.Dispose () [0x001ec] in <8bca7f503b7943c0aceeedfbf6372e05>:0 
  at CKAN.GameInstance.Scan () [0x0013d] in <79ae9046bd9c43dabfd2f7be4b883034>:0 
  at CKAN.ConsoleUI.ModListScreen.ScanForMods () [0x0000b] in <79ae9046bd9c43dabfd2f7be4b883034>:0 
  at CKAN.ConsoleUI.ModListScreen.GetAllMods (CKAN.ConsoleUI.Toolkit.ConsoleTheme theme, System.Boolean force) [0x00000] in <79ae9046bd9c43dabfd2f7be4b883034>:0 
  at CKAN.ConsoleUI.ModListScreen..ctor (CKAN.GameInstanceManager mgr, System.Boolean dbg, CKAN.ConsoleUI.Toolkit.ConsoleTheme regTheme) [0x00045] in <79ae9046bd9c43dabfd2f7be4b883034>:0 
  at CKAN.ConsoleUI.ConsoleCKAN..ctor (CKAN.GameInstanceManager mgr, System.String themeName, System.Boolean debug) [0x00081] in <79ae9046bd9c43dabfd2f7be4b883034>:0 
  at CKAN.ConsoleUI.ConsoleUI.Main_ (System.String[] args, CKAN.GameInstanceManager manager, System.String themeName, System.Boolean debug) [0x00005] in <79ae9046bd9c43dabfd2f7be4b883034>:0 
  at CKAN.CmdLine.MainClass.ConsoleUi (CKAN.GameInstanceManager manager, CKAN.CmdLine.ConsoleUIOptions opts, System.String[] args) [0x00034] in <79ae9046bd9c43dabfd2f7be4b883034>:0 
  at CKAN.CmdLine.MainClass.RunSimpleAction (CKAN.CmdLine.Options cmdline, CKAN.CmdLine.CommonOptions options, System.String[] args, CKAN.IUser user, CKAN.GameInstanceManager manager) [0x002df] in <79ae9046bd9c43dabfd2f7be4b883034>:0 
  at CKAN.CmdLine.MainClass.Execute (CKAN.GameInstanceManager manager, CKAN.CmdLine.CommonOptions opts, System.String[] args) [0x00296] in <79ae9046bd9c43dabfd2f7be4b883034>:0 
  at CKAN.CmdLine.MainClass.Main (System.String[] args) [0x000e9] in <79ae9046bd9c43dabfd2f7be4b883034>:0 
```

**Comment by jfinkbei** [comment]
I'm seeing a similar error on Arch Linux.
`System.Transactions.TransactionAbortedException: Transaction failed ---> System.ArgumentOutOfRangeException: Number must be either non-negative and less than or equal to Int32.MaxValue or -1.
Parameter name: timeout
  at System.Threading.WaitHandle.WaitOne (System.TimeSpan timeout, System.Boolean exitContext) [0x00026] in <5b78e93e8e684284bea0e5de7138cf95>:0 
  at System.Transactions.Transaction.DoPreparePhase () [0x00055] in <300541d001d949feb35b6a82c13562f2>:0 
  at System.Transactions.Transaction.DoCommit () [0x00076] in <300541d001d949feb35b6a82c13562f2>:0 
  at System.Transactions.Transaction.CommitInternal () [0x00022] in <300541d001d949feb35b6a82c13562f2>:0 
   --- End of inner exception stack trace ---
  at System.Transactions.Transaction.CommitInternal () [0x00039] in <300541d001d949feb35b6a82c13562f2>:0 
  at System.Transactions.TransactionScope.Dispose () [0x001ec] in <300541d001d949feb35b6a82c13562f2>:0 
  at CKAN.GameInstance.Scan () [0x0013d] in <e4787d5c004c42ed906f7bb77c342965>:0 
  at CKAN.Main.CurrentInstanceUpdated (System.Boolean allowRepoUpdate) [0x00015] in <e4787d5c004c42ed906f7bb77c342965>:0 
  at CKAN.Main.OnLoad (System.EventArgs e) [0x0013b] in <e4787d5c004c42ed906f7bb77c342965>:0 
  at System.Windows.Forms.Form.OnLoadInternal (System.EventArgs e) [0x00023] in <007780e2b01b40169d061cd41ace95d1>:0`

**Comment by HebaruSan** [comment]
This was fixed in #4119.
